### DISCUSSION TITLE
Need to send attachment in a POST request using Budibase REST API

### DISCUSSION THREAD

**Discussion by aashray16** [Budibase/budibase#11736] [Help] [answered]
I want to send attachments (pdf/image/ppt) files to a external rest api using POST method in Budibase. When we tried it using POSTMAN, we can send it in body as a binary format but wasnt able to do the same in Budibase. We also tried using form-data format in the body in POSTMAN but still cant replicate the same in budibase.

**Comment by melohagan** [comment, marked as answer]
Hey @aashray16 

There's an open issue for this one here: https://github.com/Budibase/budibase/issues/7007

**Reply by Abd-Atif** [reply]
@aashray16 is there any update on this?
